src/lib/loaders/loaders_without_authentication/gravity.ts

```diff
@@ -85,6 +85,9 @@ export default (opts) => {
     geneLoader: gravityLoader((id) => `gene/${id}`),
     genesLoader: gravityLoader("genes"),
     heroUnitsLoader: gravityLoader("site_hero_units"),
+    identityVerificationLoader: gravityLoader(
+      (id) => `identity_verification/${id}`
+    ),
```

Comment:
Should this be a `gravityUncachedLoader` instead? I see we almost never use that except for auction lots and system time. In this case, there's no caching benefit because any verification is specific to a single visitor.

Reply:
Seems reasonable to use `gravityUncachedLoader` here.

Reply:
Interesting! One of the main benefits of the _without_ authentication loaders is that it can cache. How about repeated calls for the same ID verification? That seems like a nice thing to subject to Metaphysics's built in throttling via cache with short TTL.

It's pretty minor, but I might stick with `gravityLoader` in this file, unless there was a need for 100% real time data, like there is for `system/time` (or some auction-related endpoints).

Reply:
Also, as an FYI, by adding this in the _without_ authentication file w/ the same name as the _with_ authentication , this will automatically use the _with_ authentication one (which passes along the access token and skips MP caching) if an access token was provided in the request.

This may not be _exactly_ the behavior you desire (it might be?). Another option is to destructure `context` in the resolver in such a way that it gives you access to _both_ the w/ and w/o authentication loaders , and then pick the specific one based on the arguments (perhaps there's one argument that requires a token and one that does not).

An example of that destructuring style is here: https://github.com/artsy/metaphysics/blob/e7f9de7620245bc14c6561e23f8c6eb501369b3f/src/schema/v2/filterArtworksConnection.ts#L478-L479 , and then selecting the right loader based on arguments: https://github.com/artsy/metaphysics/blob/e7f9de7620245bc14c6561e23f8c6eb501369b3f/src/schema/v2/filterArtworksConnection.ts#L531-L542

Reply:
That was intentional, but please advise if you think it unwise. I figured we _might_ want to return different fields eventually when an admin is consuming the data rather than a user or unauthenticated visitor.